### DISCUSSION TITLE
Multiple Domains Same Keywords Tracking

### DISCUSSION THREAD

**Discussion by syamsulalam** [towfiqi/serpbear#251] [Ideas]
got multiple domains which targeting same keywords, would be really nice to have option like domain selection box or check box below keywords input form to track it for multiple domain. so 1 scraping credit for 1 keyword can be used to track position of multiple domains. would be very helpful because as of now it takes 10 credit for 1 keyword to be tracked across 10 domains.

**Comment by EmbDclic** [comment]
Exactly, so you don't have to scrape the same SERP X times when you want to make a comparison between competitors, for example. I don’t think it’s possible right now.
